onnxruntime/core/framework/instant_op.cc

```diff
@@ -0,0 +1,273 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Why is the API implementation in the framework folder?

Reply:
Like discussed offline, let's keep in the folder but rename to standalone_op_invoker.cc.